exokit-macos

```diff
@@ -14,4 +14,4 @@ if [ -x "$DIR/node/bin/node" ]; then
 fi
 
 cd "$DIR"
-exec "$NODE_BIN" "$DIR/index.js" -h
```

Comment:
Couldn't this just be `-h "$@"`? That's what the other places use I think.

Reply:
No, that would always pass in -h.